helmfile.d/snippets/templates.gotmpl

```diff
@@ -4,6 +4,7 @@ templates:
     chart: ../charts/{{ .Release.Name }}
     values:
       - ../values/{{ .Release.Name }}/{{ .Release.Name }}.gotmpl
+      - ./snippets/rawValues.gotmpl
```

Comment:
please be aware that not all releases are deployed via `snippets/templates.gotmpl`. Are the others still reading in your new `rawValues`?